templates/registry/markdown/attribute_namespace.md.j2

```diff
@@ -39,11 +39,11 @@
 | Attribute  | Type | Description  | Examples  | Stability |
 |---|---|---|---|---|
 {%- for attribute in group.attributes | sort(attribute="name") %}
-| {{ attrs.name(attribute) }} | {{ attrs.type(attribute) }} | {{ attribute.brief | trim }}{{ notes.add(attribute.note) }} | {{ attrs.examples(attribute) | trim }} | {{ stability.badge(attribute.stability, attribute.deprecated) | trim }} |
+| <a id="{{ attribute.id | replace('.', '-') | replace('_', '-') }}" href="#{{ attribute.id | replace('.', '-') | replace('_', '-') }}">{{ attrs.name(attribute) }}</a> | {{ attrs.type(attribute) }} | {{ attribute.brief | trim }}{{ notes.add(attribute.note) }} | {{ attrs.examples(attribute) | trim }} | {{ stability.badge(attribute.stability, attribute.deprecated) | trim }} |
```

Comment:
Again, if you look at attribute_macros.j2 you will see that it uses `attribute.name`. There's no `id`. Now, all id/href are empty...

Reply:
How about now? @joaopgrassi  I apologize for the back and forth, I seemed not to be getting it before.